Lib/test/test_msvcrt.py

```diff
@@ -11,8 +13,6 @@
 import _winapi
 import msvcrt;
 
-from _testconsole import write_input, flush_console_input_buffer
```

Comment:
flush_console_input_buffer() was only used in that test. If you don't need it anymore, please remove also the associated C code

write_input(): is it used somewhere else?

Reply:
Ok, I'll remove it after current tests finished to see if this change is stable. The `write_input` is used in `test_winconsoleio` so we should keep it.

Reply:
Removed.